emsdk.py

```diff
@@ -665,6 +665,11 @@ def download_file(url, dstpath, download_even_if_exists=False, filename_prefix='
         sys.stdout.flush()
   except Exception as e:
     print("Error downloading URL '" + url + "': " + str(e))
+    # Modified code to inform solution for common "...[SSL: CERTIFICATE_VERIFY_FAILED] ..." error
+    # Error results in issues raised for similar problem: #133 #136 #140 #176 #6275 #6548 #6723 #9036  ...
+    r = "SSL: CERTIFICATE_VERIFY_FAILED"
+    if re.match(r, str(e)):
```

Comment:
these two lines can be just `if "SSL: CERTIFICATE_VERIFY_FAILED" in str(e):`

Reply:
@kripken, great hint, sure and much cleaner. Yes...been a little while since I've Pythoned with conviction. It can be such a lovely concise language. WIP as I've come across the same error on a Linux machine, so have the opportunity to work on a live error that hasn't been fixed.

Reply:
@kripken The issue has a different message on linux. I have included a similar re for the message and updated the message.  I'll rebase based on changes from yesterday and squash the pull request.